### DISCUSSION TITLE
cant make it work

### DISCUSSION THREAD

**Discussion by Artiom97es** [FlareSolverr/FlareSolverr#1164] [Q&A]
![imagen](https://github.com/FlareSolverr/FlareSolverr/assets/29014060/301d9ef8-c41f-4e57-afdf-47a6085fe39c)
im using unraid, dont know about this things, how i add v1 thing 
![imagen](https://github.com/FlareSolverr/FlareSolverr/assets/29014060/f835c4aa-7cf0-4abd-a98e-88c37686e190)

**Comment by ilike2burnthing** [comment]
Open a new issue, providing the information requested.

**Reply by Artiom97es** [reply]
sorry do you have any post about what i need to write? or provide

**Reply by ilike2burnthing** [reply]
The issue template tells you.
